conf/distro/include/ostro-xt-supported-recipes.txt

```diff
@@ -0,0 +1,3 @@
+libevent@core
+libxscrnsaver@core
```

Comment:
It is used in https://github.com/crosswalk-project/chromium-crosswalk/blob/5c3da74dd8cdf1cba7bc0d016d4ab36d689de922/ui/base/idle/idle_query_x11.cc#L7